chainerrl/misc/async.py

```diff
@@ -132,6 +133,10 @@ def set_seed_and_run(process_idx, run_func):
 
     for p in processes:
         p.join()
+        if p.exitcode < 0:
+            warnings.warn(
+                "Process {}'s exit code was not zero but {}".format(
```

Comment:
I'd like to know the process_idx.
How about `"Process #{} (pid={}) exited with nonzero status {}".format(i, p.pid, p.exitcode)`?